ella_flatcomments/models.py

```diff
@@ -178,3 +185,34 @@ def save(self, **kwargs):
         if self.submit_date is None:
             self.submit_date = timezone.now()
         super(FlatComment, self).save(**kwargs)
+
+    def has_edit_timer(self):
+        '''has_edit_timer() -> bool
+        '''
+        return EDIT_TIMER_ENABLED
+
+    def is_edit_timer_expired(self):
+        '''is_edit_timer_expired() -> bool
+        Check whether the comment is still within the allowed edit time
+        from the creation time.
+        '''
+        age = datetime.now(self.submit_date.tzinfo) - self.submit_date
+        if age >= timedelta(minutes=EDIT_TIMER_MINUTES):
+            return True
+        return False
+        
+    def get_remaining_edit_time(self):
+        '''get_remaining_edit_time() -> str
+        Returns the remaining edit time from comment creation. The returned
+        string is formatted as HH:MM:SS, e.g. 0:01:23
+        '''
+        age = datetime.now(self.submit_date.tzinfo) - self.submit_date
+        edit_time = timedelta(minutes=EDIT_TIMER_MINUTES)
```

Comment:
construct edit_time flobally and just reuse it, no need to keep reconstructing the `timedelta` object